fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/IndexingThrottle.java

```diff
@@ -196,20 +196,22 @@ <R> CompletableFuture<R> throttledRunAsync(@Nonnull final Function<FDBRecordStor
 
         AtomicInteger tries = new AtomicInteger(0);
         CompletableFuture<R> ret = new CompletableFuture<>();
-        final ExponentialDelay delay = new ExponentialDelay(common.getRunner().getDatabase().getFactory().getInitialDelayMillis(),
-                common.getRunner().getDatabase().getFactory().getMaxDelayMillis());
         AsyncUtil.whileTrue(() -> {
             loadConfig();
             return common.getRunner().runAsync(context -> common.getRecordStoreBuilder().copyBuilder().setContext(context).openAsync().thenCompose(store -> {
-                for (Index index: common.getTargetIndexes()) {
```

Comment:
Is the change to this file related to mutual indexing? At its face, it seems unrelated.

Reply:
Yes it is. 
Mutual indexer should be prepared to other processes converting index state(s) to READABLE. This is the common case.